File: Python3/1160 - Crescimento Populacional.py
T = int(input())
for i in range(T):
    PA, PB, G1, G2 = map(float, input().split())
    X, Y, CONTADOR = PA, PB, 0
    while True:
        X += int(PA * (G1 / 100))
        Y += int(PB * (G2 / 100))
        PA = X
        PB = Y
        CONTADOR += 1
        if CONTADOR > 100 or PA > PB:
            break
    if CONTADOR > 100:
        print('Mais de 1 seculo.')
    else:
        print('{} anos.'.format(CONTADOR))

File: C#/1160 - Crescimento Populacional.cs
using System; 

class URI {

    static void Main(string[] args) { 

    int CONTADOR = 0, PA = 0, PB = 0, T = int.Parse(Console.ReadLine());
    double G1 = 0, G2 = 0;
    string [] S;

    for (int i = 0; i < T; i++) {
      
      S = Console.ReadLine().Split();
      PA = int.Parse(S[0]); 
      PB = int.Parse(S[1]); 
      G1 = double.Parse(S[2]); 
      G2 = double.Parse(S[3]);

      while (PA <= PB && CONTADOR <= 100) {

        PA += (int)((PA * G1) / 100);
        PB += (int)((PB * G2) / 100);
        CONTADOR ++;

      }

      if (CONTADOR > 100)
        Console.WriteLine("Mais de 1 seculo.");
      else 
        Console.WriteLine("{0} anos.", CONTADOR);

      CONTADOR = 0;
    }

    }

}